zproject/settings.py

```diff
@@ -1179,11 +1185,25 @@ def get_secret(key: str, development_only=False) -> Optional[str]:
     'lstrip_blocks': True,
 })
 
+two_factor_template_options = deepcopy(default_template_engine_settings['OPTIONS'])
```

Comment:
Let's add this comment before this line:
`# django-two-factor uses the default Django template engine (not Jinja2), so we need to add config for it here`